src/app/fsat/explore-opportunities/explore-opportunities-form/fan-data-form/fan-data-form.component.html

```diff
@@ -127,7 +127,7 @@
     </div>
   </form>
 </li>
-<li *ngIf="showFanType || isVFD">
+<li *ngIf="fsat.modifications[exploreModIndex].exploreOppsShowFanType || isVFD">
```

Comment:
I think this should be `.hasOpportunity` otherwise it will be showing if hasOpportunity is true or false